recipes/vollseg-napari/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "vollseg-napari" %}
+{% set version = "2.3.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/vollseg-napari-{{ version }}.tar.gz
+  sha256: bc1a435f3fbd08a38f5a6fae0b7d7581ee91e56e46f2ae933f59f1d0de8fe57f
+
+build:
+  skip: true  # [py<37]
```

Comment:
Upstream README says there is a maximum python version? <3.9?

Reply:
setup.py says >=3.7 🤷